templates/views/AttitudeBarView.html

```diff
@@ -0,0 +1,28 @@
+<span>
+<div class="attitudeBarView yaw component">
+  <div class="attitudeBarView yaw block back"></div>
+  <div class="attitudeBarView yaw block front">0</div>
+  <span class="attitudeBarView block component yaw">
+    <input type="number" class="attitudeBarView input yaw"></input><button class="attitudeBarView send yaw">Set</button>
+  </span>
+</div>
+<!-- <p>Yaw Setpoint: <span class="yaw-setpoint-value">--</span> &deg;</p> -->
+<div class="attitudeBarView pitch component">
```

Comment:
You're not using classes properly. If you've got nested components they don't need to be the same class